src/js/tabs/adjustments.js

```diff
@@ -290,8 +290,10 @@ TABS.adjustments.adjust_template = function () {
 
     var selectFunction = $('#functionSelectionSelect');
     var elementsNumber;
-
-    if (semver.gte(CONFIG.apiVersion, "1.41.0")) {
+    
+    if (semver.gte(CONFIG.apiVersion, "1.43.0")) {
```

Comment:
We have started to use constants to replace the version string: API_VERSION_1_43